src/compiler/checker.ts

```diff
@@ -13201,12 +13201,13 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             const paramSymbol = createSymbol(
                 SymbolFlags.FunctionScopedVariable | (isOptional && !isRestParam ? SymbolFlags.Optional : 0),
                 paramName || `arg${i}` as __String,
+                isRestParam ? CheckFlags.RestParameter : isOptional ? CheckFlags.OptionalParameter : 0,

```

Comment:
`0` should probably be `undefined` or `CheckFlags.None`.

Reply:
I followed the existing "convention" here with that `0` :p the first argument here is passed in this very way

Reply:
Er, well, that's `SymbolFlags`, but yeah that also doesn't use `None` for some reason, boo

Reply:
I can adjust both places if you'd prefer that

Reply:
Eh, it's fine; commenting out `.None` shows 5 other places we used zero and none where we used `None`.